Changelog.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [unreleased]
+- Refactored grade entry form component to make use of assignment chart (#6118)
```

Comment:
I think this entry could be improved a bit. The high-level info for users is we're now displaying summary statistics for grade entry forms, plus that for individual columns. The fact that you've refactored the code to reduce duplication is more of an internal consideration.